apps/backend/src/applications/utils.ts

```diff
@@ -2,8 +2,20 @@ import { Application } from './application.entity';
 import { Cycle } from './dto/cycle';
 import { Semester } from './types';
 
-// TODO get the current cycle's year and semester from env variables
-export const getCurrentCycle = () => new Cycle(2024, Semester.SPRING);
+export const getCurrentSemester = (): Semester => {
+  const month: number = new Date().getMonth();
+  if (month >= 0 && month <= 5) {
+    return Semester.FALL; // We will be recruiting for the fall semester during Jan - Jun
```

Comment:
⛏️ I think it might be better to have it so that the fall recruitment cycle span from Feb to Aug and spring cycle from Sept to Jan, just bc recruitment sometimes spills into the summer or into the beginning of the next semester and there'll be bugs if the cycle changes before we wrap it up

Reply:
Makes sense, changes have been pushed!